ocaml/testsuite/tests/typing-higher-jkinds/equalities.ml

```diff
@@ -0,0 +1,110 @@
+(* TEST
+ flags = "-extension layouts_alpha";
+ expect;
+*)
+
+type t : value => value
+
```

Comment:
I think these are a little easier to read if you get rid of the blank line before the `expect` block.